ipfs-8-to-9/migration/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/ipfs/fs-repo-migrations/ipfs-8-to-9/migration
+
+go 1.13
+
+require (
+	github.com/ipfs/fs-repo-migrations v1.4.0
+	github.com/ipfs/go-cid v0.0.5
+	github.com/ipfs/go-datastore v0.3.1
+	github.com/ipfs/go-filestore v0.0.4-0.20200203163551-8cc45055f624 // indirect
+	github.com/ipfs/go-ipfs v0.4.22-0.20200130064341-6750ee973e2a
```

Comment:
TODO: link to newer after https://github.com/ipfs/go-ipfs/pull/6816 has the right dep-set and is green.